README.md

```diff
@@ -109,14 +116,55 @@ res = jsonlogic_rs.apply(
 
 assert res == True
 
-# If You have serialized JsonLogic and data, the `apply_serialized` method can 
+# If You have serialized JsonLogic and data, the `apply_serialized` method can
 # be used instead
 res = jsonlogic_rs.apply_serialized(
     '{"===": [{"var": "a"}, 7]}',
     '{"a": 7}'
 )
 ```
 
+### Commandline
+
+``` raw
+Parse JSON data with a JsonLogic rule.
+
+When no <data> or <data> is -, read from stdin.
+
+The result is written to stdout as JSON, so multiple calls
+can be chained together if desired.
+
+USAGE:
+    jsonlogic <logic> [data]
```

Comment:
```suggestion
    jsonlogic <logic> [<data>]
```

Reply:
this is auto-generated by `clap`. I think it uses `<name>` for required arguments and `[name]` for optional ones. I can see about changing it

Reply:
I see, that would technically not be docopt compliant unless [`data` was a literal input](http://docopt.org/) but I think that's no big deal if it's autogened